code/modules/industrial_lift/industrial_lift.dm

```diff
@@ -393,11 +393,12 @@ GLOBAL_LIST_EMPTY(lifts)
 					qdel(victim_machine)
 
 			for(var/mob/living/collided in dest_turf.contents)
+				var/damage_multiplier = collided.maxHealth * 0.01
 				if(lift_master_datum.ignored_smashthroughs[collided.type])
 					continue
 				to_chat(collided, span_userdanger("[src] collides into you!"))
 				playsound(src, 'sound/effects/splat.ogg', 50, TRUE)
-				var/damage = rand(9, 28) * collision_lethality
+				var/damage = rand(14, 44) * collision_lethality * damage_multiplier
```

Comment:
This change doesn't seem to be documented in your changelog?

Reply:
balance: Tram Malfunction: Damage multiplier added to account for different base health levels

Reply:
The old calculation was rand(9, 28) * 2, now it's rand(14, 44) * 1.25